web/src/features/charts/hooks/usePriceChartData.ts

```diff
@@ -30,6 +30,9 @@ export function usePriceChartData() {
     return { isLoading, isError };
   }
 
+  // We assume that if the first element has price disabled, all of them do
+  const priceDisabledReason = Object.values(zoneData.zoneStates)[0].price?.disabledReason;
+
```

Comment:
Would it not be safer if we checked if any of them has price disabled instead of just the first?

(Might be total overkill)

Reply:
This only checks the first element though, which in 99.9% of the situations should be enough but checking all values would catch 100% of the situations.

Reply:
We could and I first had a `.some()`, but then we also need to handle the error message that in theory could differ. So I figured it would not be worth adding some complexity here when we control this value internally :)